repository-data/webfiles/src/main/resources/site/freemarker/common/macro/sections/tableauLookupSection.ftl

```diff
@@ -157,6 +162,10 @@
                         var longitude = data["result"]["longitude"];
                         viz${index}Url = encodeURI("${section.url}".split("?")[0] + "?MSOA Code=" + msoa + "&Lat=" + latitude + "&Lon=" + longitude + "&Distance=" + parseInt(viz${index}Elements.distance().value) + "&Postcode=" + postcode + "&:refresh=yes");
 
+                        // build the download link and base it on div being empty
+                        downloadVizLink = encodeURI("${section.url}".split("?")[0]+".csv" + "?MSOA Code=" + msoa + "&Lat=" + latitude + "&Lon=" + longitude + "&Distance=" + parseInt(viz${index}Elements.distance().value) + "&Postcode=" + postcode + "&:refresh=yes");
+                        showPostcode = postcode;
```

Comment:
It is also possible to pass showPostcode into 'loadViz' as an argument. However it would be better to remove 'showPostcode' see the 2nd note on line 136.

Reply:
Hi, did not want to change to much code with passing arguments. But would be more than happy for you to write directly into the code, to how you visage these arguments being passed and functioning. Let me know when you have made amendments. KR